views/scss/qti/_qti-base.scss

```
@@ -217,6 +217,9 @@ $mainContainer : '.qti-item';
 
     .qti-block {
         border:1px solid transparent;
+        & img {
```

You don't need an ampersand here. I mean, it will work, but in fact that's not a correct usage of it.
```suggestion
        img {
```